operators/pkg/instctrl/containers.go

```diff
@@ -28,6 +31,60 @@ import (
 	"github.com/netgroup-polito/CrownLabs/operators/pkg/utils"
 )
 
+const (
+	UserPvcSecret              = "user-pvc-secret"
+	UserPvcSecretServerIPKey   = "server-ip"
+	UserPvcSecretServerNameKey = "server-name"
+	UserPvcSecretPathKey       = "path"
+	NfsMountPath               = "/media/MyDrive"
+	NfsVolumeName              = "nfs-vol"
+)
+
+func (r *InstanceReconciler) GetNFSVolume(ctx context.Context, volume *v1.Volume, volumeMount *v1.VolumeMount) error {
+	// Retrieve the correct Tenant from the given context
+	tenant := clctx.TenantFrom(ctx)
+	if tenant == nil {
+		retErr := errors.New("unable to retrieve tenant from the context")
+		klog.Errorf("%s", retErr)
+		return retErr
+	}
+
+	if tenant.Status.PersonalNamespace.Created {
```

Comment:
Following the previous comment: I'd retrieve the secret directly from the same namespace of the instance, since it should already be in the same namespace of the secret.

Reply:
Ok, resolved